meta-sokol-flex-distro/dynamic-layers/openembedded-layer/recipes-support/lvm2/lvm2_%.bbappend

```diff
@@ -3,6 +3,6 @@
 # ---------------------------------------------------------------------------------------------------------------------
 
 FLEX_LVM_PKG = ""
-FLEX_LVM_PKG:sokol-flex = "recipes-support/lvm2/lvm2-flex.inc"
+FLEX_LVM_PKG:sokol-flex = "lvm2-flex.inc"
 
 include ${FLEX_LVM_PKG}
```

Comment:
you should use require here instead of include. it will make sure the file is added when needed and we won't have such problems in the future
cc @abelal